topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -0,0 +1,523 @@
+- What steps are required to prepare single-cell data (e.g., clustering, annotation, and metadata addition) for pseudobulk analysis?  
+- How can we use pseudobulk data prepared with Decoupler to perform differential expression analysis using edgeR in Galaxy?  
+objectives:
+- Understand the principles of pseudobulk analysis in single-cell data
+- Understand and generate the pseudobulk expression matrix with Decoupler
+- Perform differential expression analysis using edgeR
+time_estimation: 3H
+key_points:
+- Pseudobulk analysis approach bridges the gap between single-cell and bulk RNA-seq data  
+- Decoupler tool generates a pseudobulk count matrix, enabling downstream differential expression and functional analyses 
+- edgeR is a robust tool for differential expression in pseudobulk datasets  
+contributors:
+- dianichj
+- pavanvidem
+tags:
+- single-cell
+- transcriptomics
+- pseudobulk
+
+---
+
+Pseudobulk analysis is a powerful technique that bridges the gap between single-cell and bulk RNA-seq data. It involves aggregating gene expression data from groups of cells within the same biological replicate, such as a mouse or patient, typically based on clustering or cell type annotations ({% cite Murphy2022 %}).
+
+A key advantage of this approach in differential expression analysis is that it avoids treating individual cells as independent samples, which can underestimate variance and lead to inflated significance or overly optimistic p-values ({% cite Squair2021 %}). This occurs because cells from the same biological replicate are inherently more similar to each other than cells from different samples. By grouping data into pseudobulk samples, the analysis aligns with the experimental design, as in bulk RNA-seq, leading to more reliable and robust statistical results ({% cite Murphy2022 %}).
+
+Beyond enhancing statistical validity, pseudobulk analysis enables the identification of cell-type-specific gene expression and functional changes across biological conditions. It balances the detailed resolution of single-cell data with the statistical power of bulk RNA-seq, providing insights into the functional transcriptomic landscape relevant to biological questions. Overall, for differential expression analysis in multi-sample single-cell experiments, pseudobulk approaches demonstrate superior performance compared to single-cell-specific DE methods ({% cite Squair2021 %}). 
+
+In this tutorial, we will guide you through a pseudobulk analysis workflow using the **Decoupler** and **edgeR** tools available in Galaxy ({% cite Badia-iMompel2022 %}) ({% cite Liu2015 %}). These tools facilitate functional and differential expression analysis, and their output can be integrated with other Galaxy tools to visualize results, such as creating Volcano Plots, which we will also cover in this tutorial.
+
+> <agenda-title>Pseudobulk Analysis Pipeline Agenda</agenda-title>
```

Comment:
the agenda will be automatically generated based on your section titles, pleas replace this with 


```
> <agenda-title></agenda-title>
>
> In this tutorial, we will cover:
>
> 1. TOC
> {:toc}
>
{: .agenda}
```